packages/video_player/video_player/lib/video_player.dart

```diff
@@ -283,12 +289,16 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
   /// Only set for [asset] videos. The package that the asset was loaded from.
   final String? package;
 
+  Future<ClosedCaptionFile>? _closedCaptionFileFuture;
+
   /// Optional field to specify a file containing the closed
   /// captioning.
   ///
   /// This future will be awaited and the file will be loaded when
   /// [initialize()] is called.
-  final Future<ClosedCaptionFile>? closedCaptionFile;
+  Future<ClosedCaptionFile>? get closedCaptionFile {
```

Comment:
Please put this after the field declarations, not in the middle of them. Right before `setClosedCaptionFile` would be the best place.